CHANGELOG.md

```diff
@@ -47,6 +47,7 @@
 - Added support for `Series.str.lstrip` and `Series.str.rstrip`.
 - Added support for `DataFrameGroupby.size` and `SeriesGroupby.size`.
 - Added support for `DataFrame.expanding` and `Series.expanding` for aggregations `count`, `sum`, `min`, `max`, `mean`, `std`, and `var` with `axis=0`.
+- Added support for `Series.case_when` except when condition or replacement is callable.
```

Comment:
Nit: Can we add this at the end of the new features section?

Reply:
Done